tests/utils/misc/misc.go

```diff
@@ -23,7 +23,7 @@ import (
 
 // SleepForSec sleep for a given number of seconds
 func SleepForSec(sec int) {
-	log.Printf("Sleep for %d seconds", sec)
+	//log.Printf("Sleep for %d seconds", sec)
```

Comment:
These messages for some of the tests are creating a lot of logs. I'll uncomment before merge.